lore/species/tajaran/The Great Frost.md

```diff
@@ -0,0 +1,85 @@
+
+Decades passed before each innovation saw its full potential but with it the normalization of life had come. The struggles with the Shards of Messa brought more strife but even that passed with time. If left to their own devices the problems of the tajarans would have slowly been resolved. Through luck outside intervention came and with it the easing of their strife.
+
+Modern Day
+
+With the aid of outside groups supplying the holds with rare materials they could not produce, the creation of more hearth engines, the success of the arboretums, the frost breakers, and the now well armed and hardened populace united under S�randarr life became worth living. Much of the focus on many tajarans has been brought under the preservation of old traditions and expanding into the world with the making of new hearth engines. Priests guided by Sun Wardens traveled to old temples, towns, and cities searching for ancient scrolls of S�randarr. Artwork, writings, and any artifact of significance was taken and brought back to the hearth engines to once again be preserved in the light of S�randar�s grace. The bodies found were given funeral rites and the dead properly dealt with out of respect. With all eyes turned towards these noble goals civil strife was rare, unity prevailed.
+
+Nerahni
+The Nerahni once again took up their tradition of art and creation. The great frost had killed many of them and the survivors found purpose in expression. Though much of their art became grim and macabre their was a hope in what they made that effected all who saw it. The design of structures, temples, frost breakers, and the paintings they made decorate much of the holds. Those among them who did not become artist founds themselves pushed into practical positions with the creation of architecture and stone masonry before they slowly reclaimed their former lofty position.

```

Comment:
"... their art became grim and macabre, **there** was a hope in what they made that **a**ffected all..."
"... among them who did not become artist**s** found themselves pushed..."